### ISSUE TITLE
Don't know AtoX? You may have to be pushed on the beach by the tide

### ISSUE THREAD

**Issue by TONGZHENGSHIJIE** [TONGZHENGSHIJIE/-#89]
The blockchain 4.0 era led by AtoX has arrived. If you haven't heard of the blockchain, I'm afraid it's not just being pushed on the beach by the tide.

Blockchain is a new thing in the new era. For blockchain, even people who are familiar with it may not really understand what the blockchain is.

By definition, a blockchain is a distributed storage method. If it is more popular, it is called distributed accounting.

What is distributed accounting? To put it simply, I borrowed ten yuan from you, and I made a loan with you. After that, when you take this loan and ask me for money, I must pay for it. But if this loan is lost, I will not admit it when you ask me for money. What should you do? The so-called blockchain distributed accounting, that is, I borrowed ten yuan from you, I will give you a loan, and at the same time make a few more copies of the loan, and give it to someone else, so that even if your loan is lost, they can testify for you, this is the distributed accounting of the blockchain.
The concept of a blockchain is so simple, but the technology behind it is very complex. For example, storage, peer-to-peer transmission, encryption algorithms and traceability are very complex and very deep-seated.

It is these complex technologies that have brought about a revolutionary change on a global scale, covering transfers, remittances, cross-border settlements, intelligent bills, securities and investment markets, smart payment scenarios, and even had a profound impact on legal, logistics, food traceability, retail and other industries.
AtoX is a fourth-generation blockchain-based public blockchain project with a carefully selected trusted supernode, the highest-end RPCA protocol, and achieve nearly unlimited transaction speed through a cross-chain lightning network.
Its technical characteristics are reflected in the underlying architecture, distributed control structure, RPCA algorithm, etc.; technological innovation has reached an unprecedented height, which is reflected in the ten aspects such as historical records cannot be tampered with, data is flexible and recoverable, supports multiple transaction modes, private key separation, strict mode, chain compression, audit, dump, subscription, cross-chain and so on.
The development of AtoX is a summary and elevation of past technological achievements, and provides a reference for future project development.

In short, the blockchain is fierce and will have a huge impact on our production and life. Only by complying with the trend of the times, so  we can not be abandoned by the times.
Choose the AtoX project to let you break out in 2019. Participation link: 
Atox Project official website: http://www.atoxchain.io
Atox project Chinese community: https://t.me/axc_china 
Atox project English community: https://t.me/axc_english
